code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -74,6 +74,51 @@
 /obj/item/weapon/cautery/is_hot()
 	return 1
 
+/obj/item/weapon/cautery/laser
+	sharpness = 1.5
+	sharpness_flags = SHARP_TIP | SHARP_BLADE | HOT_EDGE
+	damtype = "fire"
+	force = 10.0
+	throwforce = 5.0
+	throw_speed = 3
+	throw_range = 5
+	attack_verb = list("attacks", "slashes", "stabs", "slices", "tears", "rips", "dices", "cuts")
+	hitsound = "sound/weapons/bladeslice.ogg"
```

Comment:
Doesn't make sense for a laser.

Reply:
I will add the burn sound here in place when ai change the other issue.